CHANGELOG.md

```diff
@@ -67,6 +67,7 @@
 * Fixed #3041
 * Fixed #3036
 * Possibly fixed #2927
+* Fixed #2964
```

Comment:
For future pull requests: Rather leave the changelog maintenance up to us as it can lead to very much avoidable and unnecessary merge conflicts between pull requests if everyone updates it.

Reply:
I removed it, I just assumed I always forgot to add it and you had to do it afterwards :smile: